packages/clay-modal/src/__tests__/IncrementalInteractions.tsx

```diff
@@ -209,7 +209,14 @@ describe('ModalProvider -> IncrementalInteractions', () => {
 		cleanup();
 	});
 
+	const originalError = console.error;
 	beforeAll(() => {
+		console.error = (...args: any) => {
+			if (/Warning.*not wrapped in act/.test(args[0])) {
+				return;
+			}
+			originalError.call(console, ...args);
+		};
```

Comment:
Why this? Can you just wrap the test in `act` as the error message indicates, rather than suppressing the message? There were some issues with `act` prior to React v16.9.0 but I think they've all been fixed now.

Reply:
There might be a clue in one of these:

- https://github.com/testing-library/react-hooks-testing-library/issues/14
- https://github.com/testing-library/react-testing-library/issues/281